src/modules/sensors/sensors.cpp

```diff
@@ -665,7 +665,7 @@ void Sensors::Run()
 
 	// keep adding sensors as long as we are not armed,
 	// when not adding sensors poll for param updates
-	if (!_armed && hrt_elapsed_time(&_last_config_update) > 500_ms) {
+	if (!_armed && hrt_elapsed_time(&_last_config_update) > 1000_ms) {
```

Comment:
was this just trial and error?

Reply:
After I found that it was a timing problem and did not get discovered, yes.